examples/c/example.c

```diff
@@ -763,17 +772,196 @@ AranyaError run(Team *t) {
     return err;
 }
 
+AranyaError aranya_allocate_label(AranyaClient* client, AranyaTeamId* id,
+                                  const char* label_name,
+                                  AranyaLabelId* label_id, int num_peers, ...) {
+    AranyaError err = ARANYA_ERROR_OTHER;
+
+    err = aranya_create_label(client, id, label_name, label_id);
+    EXPECT("error creating label", err);
+
+    va_list args;
+    va_start(args, num_peers);
+
+    for (int i = 0; i < num_peers; i++) {
+        AranyaDeviceId* device_id = va_arg(args, AranyaDeviceId*);
+        // va_arg promotes smaller primitives to int, so we downcast back
+        AranyaChanOp op = va_arg(args, int);
+
+        err = aranya_assign_label(client, id, device_id, label_id, op);
+        EXPECT("error assigning label", err);
+    }
+
+exit:
+    va_end(args);
+    return err;
+}
+
+// Run the AFC example.
+AranyaError run_afc_example(Team* t) {
+    AranyaError err = ARANYA_ERROR_OTHER;
+
+    unsigned char* ciphertext = NULL;
+    unsigned char* plaintext  = NULL;
+    size_t ciphertext_len;
+    size_t plaintext_len;
+
+    AranyaAfcChannelType chan_type;
+    AranyaAfcSeq seq;
+    const uint8_t* ptr;
+    size_t len;
+
+    Client* operator= & t->clients.operator;
+    Client* membera = &t->clients.membera;
+    Client* memberb = &t->clients.memberb;
+
+    AranyaLabelId label1_id;
+    AranyaChanOp op = ARANYA_CHAN_OP_SEND_RECV;
+    err = aranya_allocate_label(&operator->client, &t->id, "label1", &label1_id,
+                                2, &membera->id, op, &memberb->id, op);
+    if (err != ARANYA_ERROR_SUCCESS) {
+        goto exit;
+    }
+
+    err = aranya_sync_now(&membera->client, &t->id, sync_addrs[OPERATOR], NULL);
+    EXPECT("error calling `sync_now` to sync with peer", err);
+
+    err = aranya_sync_now(&memberb->client, &t->id, sync_addrs[OPERATOR], NULL);
+    EXPECT("error calling `sync_now` to sync with peer", err);
+
+    AranyaAfcChannel channel_a;
+    AranyaAfcCtrlMsg message;
```

Comment:
Calling this `message` could be confusing since the example also sends the `afc_msg` data message.
```suggestion
    AranyaAfcCtrlMsg ctrl;
```
Or:
```suggestion
    AranyaAfcCtrlMsg ctrl_msg;
```

Reply:
I'm still not convinced "control message" is the best descriptor of "hey, give this to the peer you want to talk with so they can create a channel", it's way too vague